extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -436,9 +435,10 @@ export class AmpAd3PImpl extends AMP.BaseElement {
    * @private
    */
   getFullWidthHeight_(width, maxHeight) {
+    // TODO(google a4a eng): remove this once adsense switches fully to
+    // fast fetch.
     if (this.element.getAttribute('data-auto-format') == ADSENSE_MCRSPV_TAG) {
```

Comment:
Should we guard this with type = 'adsense' check to ensure it only applies to AdSense traffic?  I realize this will be enforced by the validator but it could be a good thing to add for canonical pages